website/docs/filter-group/advanced-filters/advanced-filters.md

```diff
@@ -5,153 +5,149 @@ tabs: Design('advanced-filters'), Example('advanced-filters-code')
+| Active      | ![filter active state](static/advanced-filter.png) |
 
-- The filter expands in the accordion, shifting down the page content.
-- The accordion should have a fixed width.
-- When applying any filter in the trigger, show a counter with the number of applied filters.
-- Usually this group of filters has no submit buttons like "Apply", "Clear all", "Cancel".
+- Dynamic filter application (no "Apply" button) is useful when loading data while applying the filter strongly affects the output. It is important to show the content to the user immediately when applying the filter, without waiting for confirmation from the "Apply" button.
+- Filters are valid only within one report. However, they should be saved when returning to this report (within the same product) and when reloading the page.
+- When changing the base, date, or switching between reports, the values in advanced filters should be saved.
+- Filter settings are forwarded to the URL.
 
-| Case                                  | Appearance example                                            |
-| ------------------------------------- | ------------------------------------------------------------- |
-| Dynamically applied filter            | ![filter in accordion](static/advanced-filters-accordion.png) |
-| Filter applied by the "Apply" button. | ![filter in accordion](static/advanced-filters-apply.png)     |
+### Clearing a filter group
 
-### Filled filter
+- The delete button appears in the filter form after applying filters and clears the entire group of filters.
+- The advanced filters-dependent page content is reloaded and overrides the applied values.
+- By clicking the "Clear all" button, everything applied in the group will be deleted.
 
-After applying the filter:
+### Loading
+
+When the filter's content or some of the filters are loading, show [Spin](/components/spin/spin) with the "Loading..." text.
 
-- show the counter of applied filters in the trigger;
-- the counter must be recalculated after each filter application.
+![](static/loading.png)
 
-![filter in accordion](static/advanced-filters-accordion-filled.png)
+Wrap the filter form in a [SpinContainer](/components/spin-container/spin-container) when the user-selected data needs time to be applied.
 
-## Margins and styles recommendations
+![](static/advanced-filters-loading.png)
 
-If you need to increase or decrease the margins between filters inside the form, make them multiples of 4: 8px, 12px, 24px, 32px, etc.
+### Nothing found
+
+Show an ["empty" state](/components/widget-empty/widget-empty) with the option to update the filter.
 
-![filter paddings](static/margins.png)
+![](static/nothing-found-advanced-filters.png)
 
-Always visually separate filter triggers from the filters themselves. For example, use `border: 1px solid var(--border-primary)`.
+## Tooltip
 
-![filter border](static/advanced-filters-dropdown2.png)
+If the accordion needs a Close button, then show a tooltip with the text: 'Hide".
```

Comment:
Sounds like a good idea. I was thinking of adding this information to the Filter common rules page since all these filters have this button.

Reply:
By the way, I think it would be better to add a link to the FilterTrigger documentation, as it has the default hint. We haven't updated it yet according to the latest accessibility fixes, so it will be the one place with the hint text for the FilterTrigger's remove button (you can see the text on the screenshot that we didn't discuss with UX writers yet, heh). 

For now I would just add a link to the FilterTrigger section about its states. And that's it.

<img width="776" alt="image" src="https://github.com/user-attachments/assets/230aa686-e0bb-4db3-9bb4-d8d644a7f1f3">